### PR TITLE
Feat/1205335958695279 forked runtime upgrade

### PR DESCRIPTION
**Pull request by sfffaaa**
Add the script for generating the forked chain

### CODE REVIEW

File: forked.generated.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+SOURCE_PATH=${SOURCE_PATH:-"/home/jaypan/Work/peaq/fork-test/fork-binary/peaq-dev-v06042023"}
```
Comment (neutrinoks) [inline forked.generated.sh]: Is this generic? Looks like it may only work on your computer Jay...

File: forked.generated.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ln -sf $SOURCE_PATH/peaq-node data/binary
+
+# 2. Put the raw parachain file you can control here, for example: the sudo user, blah blah blah
+./data/binary build-spec --disable-default-bootnode --chain $SOURCE_PATH/parachain.plaintext.config --raw > data/fork.json
+./data/binary build-spec --disable-default-bootnode --chain $SOURCE_PATH/parachain.plaintext.config --raw > data/genesis.json
+
+# Run
+HTTP_RPC_ENDPOINT=$RPC_ENDPOINT \
+ALICE=$ALICE \
+scripts/docker-start.sh
```
Comment (neutrinoks) [inline forked.generated.sh]: Which script is this here? `docker-start.sh`?

**Review by neutrinoks — COMMENTED** [review]
There are two questions left, have a look :blush:

File: forked.generated.sh (hunk 1 above)
Comment (sfffaaa) [reply forked.generated.sh]: Although it's a default path, let me remove it and update the README

File: forked.generated.sh (hunk 2 above)
Comment (sfffaaa) [reply forked.generated.sh]: This script is in the project natively, but you are right, let me move part of them into my script, it is clearer

**Review by neutrinoks — APPROVED** [review]

**Pull request merged**